monai/bundle/utils.py

```diff
@@ -105,19 +105,42 @@
     "handlers_id": DEFAULT_HANDLERS_ID,
     "configs": {
         "tracking_uri": "$@output_dir + '/mlruns'",
+        "experiment_name": "monai_experiment",
+        "run_name": "None",
```

Comment:
According to my previous comment, I think it should be `None` instead of string "None".

Thanks.